### PR TITLE
Creating an issue template for icon submissions and updating contribution guidelines

### PR DESCRIPTION
**Pull request by jonrohan**
Updating the contribution guidelines a bit to send submissions to an issue. Also updating the contribution language around this process.

@primer/octicons

### CODE REVIEW

**Comment by jonrohan** [comment]
@ashygee This should go into #368

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -25,40 +25,44 @@ Make sure your duplicate file can be viewed by others. In the share menu "Anyone
...
+### 2. Join any separate paths
+
+Any separate shapes under the icon component need to be unioned into one path. This is for performance when serving the icons.
 
-Save the contributor’s Figma file as a `.fig` file and open in Figma (it will automatically be imported into your Drafts)
+![image](https://user-images.githubusercontent.com/54012/72647901-7c9a7e80-392e-11ea-9e9b-08c4b8e1af0c.png)
 
 
-## Step 2, Import:
+### 3. Create and/or publish the icon as a figma component
```
Comment (ashygee) [inline .github/CONTRIBUTING.md]:
```suggestion
#### 3. Create component of icon (if not already a component)
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -25,40 +25,44 @@ Make sure your duplicate file can be viewed by others. In the share menu "Anyone
...
 
-Drag and drop, or paste the updated or new icon from the contributer's file into the main Octicons Figma file. Make sure that the SVG only contains a single `path` and that it has relevant component keywords in Figma.
+Make sure you publish the icon changes to the figma component library. https://help.figma.com/hc/en-us/articles/360038665934
```
Comment (ashygee) [inline .github/CONTRIBUTING.md]:
```suggestion
If the icon is not already a component you must [create a new component](https://help.figma.com/hc/en-us/articles/360038665934). To do so, click the `Create Component` button in the top toolbar inside of Figma to add the icon as a component of the Octicons library. 

If the icon is already a component, make sure you are updating the contents of the related component.
```

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -25,40 +25,44 @@ Make sure your duplicate file can be viewed by others. In the share menu "Anyone
...
 
+### 4. Add keywords to the icon
 
-### Step 3, Add:
+Each icon needs keywords to help build out the icon search. Add `keywords: foo, bar` to the "Component" section in the figma toolbar.
 
-Once the new changes have been added the new components to the master Octicons file, you can publish to the team library in Figma.
+![image](https://user-images.githubusercontent.com/54012/72647771-26c5d680-392e-11ea-9167-462d5bb27936.png)
```
Comment (ashygee) [inline .github/CONTRIBUTING.md]:
```suggestion
![image](https://user-images.githubusercontent.com/54012/72647771-26c5d680-392e-11ea-9167-462d5bb27936.png)

#### 5. Publish changes in Figma

Once all of the changes have been made to the Octicons file, be sure to [publish the changes](https://help.figma.com/hc/en-us/articles/360038682574-Publish-Changes-to-a-Team-Library) to the component library.
<img width="449" alt="Screenshot 2020-01-17 14 43 01" src="https://user-images.githubusercontent.com/10384315/72651266-b4f28a80-3937-11ea-92db-ae51eb99c68e.png">
```

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -25,40 +25,44 @@ Make sure your duplicate file can be viewed by others. In the share menu "Anyone
...
+![image](https://user-images.githubusercontent.com/54012/72647901-7c9a7e80-392e-11ea-9e9b-08c4b8e1af0c.png)
 
 
-## Step 2, Import:
+### 3. Create and/or publish the icon as a figma component
 
-Drag and drop, or paste the updated or new icon from the contributer's file into the main Octicons Figma file. Make sure that the SVG only contains a single `path` and that it has relevant component keywords in Figma.
+Make sure you publish the icon changes to the figma component library. https://help.figma.com/hc/en-us/articles/360038665934
 
+### 4. Add keywords to the icon
```
Comment (ashygee) [inline .github/CONTRIBUTING.md]:
```suggestion
#### 4. Add keywords to the icon
```

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -25,40 +25,44 @@ Make sure your duplicate file can be viewed by others. In the share menu "Anyone
...
 Once submitted changes have been agreed upon, these instructions will guide core primer team members in merging in those changes.
 
-### Step 1, Save:
+## Adding the icon
+
+### 1. Paste into master figma file
+
+If the icon is being updated, you'll need to keep the existing component and copy the new path into it.
+
+### 2. Join any separate paths
```
Comment (ashygee) [inline .github/CONTRIBUTING.md]:
```suggestion
#### 2. Join any separate paths
```

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -25,40 +25,44 @@ Make sure your duplicate file can be viewed by others. In the share menu "Anyone
...
 ---
 
 ## Maintainers accepting changes
 
 Once submitted changes have been agreed upon, these instructions will guide core primer team members in merging in those changes.
 
-### Step 1, Save:
+## Adding the icon
+
+### 1. Paste into master figma file
```
Comment (ashygee) [inline .github/CONTRIBUTING.md]:
```suggestion
#### 1. Paste into master figma file
```

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -25,40 +25,44 @@ Make sure your duplicate file can be viewed by others. In the share menu "Anyone
...
-[<img src="https://user-images.githubusercontent.com/54012/37811102-45ec2abc-2e15-11e8-8c1d-2d162ddcdad2.png" width="700"/>](https://user-images.githubusercontent.com/54012/37811102-45ec2abc-2e15-11e8-8c1d-2d162ddcdad2.png)
 
 ---
 
 ## Maintainers accepting changes
 
 Once submitted changes have been agreed upon, these instructions will guide core primer team members in merging in those changes.
 
-### Step 1, Save:
+## Adding the icon
```
Comment (ashygee) [inline .github/CONTRIBUTING.md]:
```suggestion
### Adding the icon
```

**Pull request merged**